tests/jsonify.test/runit

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+bash -n "$0" | exit 1
+
+dbnm=$1
+
+set -e
+
+# Make sure that all queries go to the same node.
```

Comment:
Why?

Reply:
Sorry. It is a copy/paste error. I will remove.